app/controllers/appointments_controller.rb

```diff
@@ -1,6 +1,6 @@
 class AppointmentsController < ApplicationController
   before_action :set_appointment, only: [:show, :update, :destroy]
-  before_action :authenticate_user!
+  # before_action :authenticate_user!
```

Comment:
don't forget to turn this back on.   you can add to this PR by creating a new commit in this branch and pushing to github.  Then ping me by using the "@" symbol, so I get an email notification.